runtime/server_http_response.go

```diff
@@ -281,3 +281,8 @@ func (res *ServerHTTPResponse) writeBytes(bytes []byte) {
 func (res *ServerHTTPResponse) GetPendingResponse() ([]byte, int) {
 	return res.pendingBodyBytes, res.pendingStatusCode
 }
+
+// ResponseHeaders returns the underlying http response's headers
+func (res *ServerHTTPResponse) ResponseHeaders() http.Header {
```

Comment:
nitpick: `Headers` should be sufficient since the method in defined on response.